corsheaders/checks.py

```diff
@@ -10,79 +10,79 @@ def check_settings(app_configs, **kwargs):
     errors = []
     if not isinstance(corsheaders_settings.CORS_ALLOW_HEADERS, collections.Sequence):
         errors.append(
-            checks.Warning(
+            checks.Error(
                 "CORS_ALLOW_HEADERS must be sequence",
-                id="django-cors-headers.W001"
+                id="corsheaders.W001"
             )
         )
     if not isinstance(corsheaders_settings.CORS_ALLOW_METHODS, collections.Sequence):
         errors.append(
-            checks.Warning(
+            checks.Error(
                 "CORS_ALLOW_METHODS must be sequence",
-                id="django-cors-headers.W002"
+                id="corsheaders.W002"
             )
         )
     if not isinstance(corsheaders_settings.CORS_ALLOW_CREDENTIALS, bool):
         errors.append(
-            checks.Warning(
+            checks.Error(
                 "CORS_ALLOW_CREDENTIALS must be bool",
-                id="django-cors-headers.W003"
+                id="corsheaders.W003"
             )
         )
     if not isinstance(corsheaders_settings.CORS_PREFLIGHT_MAX_AGE, numbers.Integral):
```

Comment:
It might be more sane to use `int` here.

Also you could check that it's `>= 0`, since a negative age doesn't make sense.